cpp/config/suites/c/correctness

```diff
@@ -23,6 +23,7 @@
 + semmlecode-cpp-queries/Likely Bugs/Arithmetic/BadAdditionOverflowCheck.ql: /Correctness/Common Errors
 + semmlecode-cpp-queries/Likely Bugs/NestedLoopSameVar.ql: /Correctness/Common Errors
 + semmlecode-cpp-queries/Likely Bugs/UseInOwnInitializer.ql: /Correctness/Common Errors
++ semmlecode-cpp-queries/Likely Bugs/RedundantNullCheckSimple.ql: /Correctness/Common Errors
```

Comment:
I'd prefer not to add this to a suite if we aren't ready to run it in production. I'm not sure if anybody is using the correctness suite,  but I know of a 1.19 deployment where security suites are in active use.

Reply:
Point taken. I've rebased so we're no longer adding to `correctness` but instead creating a new suite named `experimental`.